_events/sr2025/kickstart.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Kickstart
+date: 2024-10-12 11:00:00 Europe/London
+end_date: 2024-10-12 17:00:00 Europe/London
+layout: event
+type: kickstart
+location: Southampton University
+---
+
+Kickstart is the event which kicks off a competition year. The structure of the
+competition, the game rules will be announced and of course any questions you
+have will be answered.
+
+Throughout the day we'll have Blueshirts (our awesome volunteers!) around in
+person, ready to answer any questions you may have and to help you through the
+microgames. These microgames will help your team become familiar with the Kit
+you've been provided and should kickstart some ideas for your robot's design.
+
+If you are unable to attend Kickstart, please [let us know][teams-contact].
+
+## Schedule
+
+_To be announced._
+
+## What to bring
+
+* Your robot kit (which will have been shipped to you beforehand)
```

Comment:
There are many reasons this may not happen

```suggestion
* Your robot kit (which should have been shipped to you beforehand)
```

Reply:
Hrm, this is complicated by this being read both before and after we're telling people about this plan :thinking:

Reply:
https://github.com/srobo/website/pull/578/commits/82d8059ad20b15b0cbaa80de82dca15050370703